README.md

```diff
@@ -1,47 +1,99 @@
 # Mojito Snowplow Storage
 
-Snowplow Storage is comprised of two key areas:
+Simple experiment events & data models to power analysis in [Mojito R Analytics](https://github.com/mint-metrics/mojito-r-analytics) and other tools. 
 
-1. ```./events```: Self-describing events emitted from experiments
-2. ```./redshift-datamodels```: SQL data models for attributing ```conversions``` back to treatment ```exposures```
+There are two parts to this repository:
+
+1. **Events** (```./events```): Self-describing events emitted from experiments
+2. **Data models** ```./redshift-datamodels```: SQL data models for attributing ```conversions``` back to treatment ```exposures``` in reporting
+
+
+## Prerequisites
+
+ - Snowplow running with Redshift as a storage target
+ - Snowplow's SQL Runner app
 
 
 ## Getting started
 
-You'll need to add three things to your Enrichment pipeline:
+[Add the events to your Iglu](https://discourse.snowplowanalytics.com/t/introductory-guide-to-creating-your-own-self-describing-events-and-contexts-tutorial/1377) and [setup your data modelling steps in SQL Runner](https://github.com/snowplow/sql-runner/wiki/Guide-for-analysts):
 
 1. [Add JSON schema to your Iglu](./events/jsonschema)
-2. [Place the JSON paths files in your JSON paths folder](./events/jsonpaths)
-3. [Create the tables in Redshift for loading shredded events](./events/sql)
-4. [Setup SQL Runner to load your report tables each day](./redshift-datamodels)
+    - This is required for event validation and shredding during enrichment
+2. (Redshift only) [Put the JSON paths files in your JSON paths folder](./events/jsonpaths)
+    - If running Redshift, this maps the JSON keys to your table fields
+3. (Redshift only) [Create the tables in Redshift for loading shredded events](./events/sql)
+    - If running Redshift, this is the table definition that shredded events will populate
+4. Setup SQL Runner to load your report tables each day ([Redshift data models](./redshift-datamodels))
+    - This step creates the data model used for reporting
+
+
+## Mojito's three core events
+
+For most experiment reports, we're interested in just three events:
+
+1. Exposures (or assignments)
+2. Failures (errors in experiment treatments, shared code or triggers)
```

Comment:
I've been using the terminology "variants" rather than "treatments" where possible, as another way to refer to or describe recipes. Not sure if we need some more consistency across our docs with this term.

Reply:
True... something we need to decide on for consistency. I guess variants lets you group up the control group too in a sense.